imagetest/test_suites/metadata/setup.go

```diff
@@ -1,17 +1,52 @@
 package metadata
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
 )
 
 // Name is the name of the test package. It must match the directory name.
 var Name = "metadata"
 
+const (
+	startupScriptTemplate = `#!/bin/bash
+echo "%s" > %s`
+	startupOutputPath = "/startup_out.txt"
+	startupContent    = "The startup script worked."
+	startupMaxLength  = 32768 // max shutdown metadata value
```

Comment:
Can you add a link to where this number is defined? Is it a constant imposed by GCE?

Reply:
Yes. https://cloud.google.com/compute/docs/metadata/setting-custom-metadata#limitations